config/config.go

```diff
@@ -1302,6 +1320,40 @@ func (re Regexp) MarshalYAML() (interface{}, error) {
 	return nil, nil
 }
 
+// RemoteQueueConfig is the configuration for the queue used to write to remote
+// storage.
+type RemoteQueueConfig struct {
```

Comment:
This struct is missing the YML field annotations that all the other config structs have.